human-guesses.js

```diff
@@ -0,0 +1,70 @@
+//The computer will pick a number and the user will guess it
+//The computer will tell the user higher or lower or that they're correct
+
+/**********Boiler Plate Begin ************/
+const readline = require("readline");
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    readlineInterface.question(questionText, resolve);
+  });
+}
+/**********Boiler Plate End ************/
+
+//Picks a random integer using the min and max values given to the function
+function randomInteger(min, max) {
+  let range = max - min + 1;
+  return min + Math.floor(Math.random() * range);
+}
+
+start();
+
+async function start() {
+  let userGuesses = [];
+  let tries = 1;
+  let minNumber;
+  let maxNumber;
+  //let stringMin = await ask("What would you like the lower limit to be? ");
+  //let min = parseInt(stringMin);
```

Comment:
If these lines (28-31) or no longer necessary consider removing them altogether to improve readabillity